connect/routes/fhir.py

```diff
@@ -99,7 +98,7 @@ async def post_fhir_data(
             message=message,
             origin_url="/fhir/" + resource_type,
             certificate_verify=settings.certificate_verify,
-            data_format=data_format,
+            data_format="FHIR",
```

Comment:
Should we explicitly indicate the version, e.g. "FHIR-R4"?  Right now, we only accept FHIR-R4 so FHIR has to mean R4.  If we decided to add R3 and to on-the-fly conversion to R4, we'd still store R4, so perhaps there's not a case for versioning within the data format type, at least for FHIR.

Reply:
Great question @ccorley ! I'll add that back in. Based on the format we "should" be able to support

FHIR-R4
X12-5010

I'll make those changes and will take a look at how we can enhance the EDI project's classification/analysis to provide these values.